components/Login/index.js

```diff
@@ -0,0 +1,85 @@
+import { Form, Input, Button, Checkbox, Typography } from "antd";
+import { MailOutlined, LockOutlined } from "@ant-design/icons";
+import { useAuth } from "~/components/Auth";
+import Link from "next/link";
+
+import styles from "./style.module.css";
+
+function LoginForm() {
+  const { Title, Text } = Typography;
+  const { errors, isLoading, login } = useAuth();
+  // TODO: Session must be remembered when 'remember' is true
+  const onFinish = ({ email, password /*, remember*/ }) => {
+    login({ email, password });
+  };
+
+  return (
+    <Form
+      name="login"
+      className={styles["login-form"]}
+      initialValues={{
+        remember: true,
+      }}
+      onFinish={onFinish}
+    >
+      <Title>Iniciar sessão</Title>
+
+      <Form.Item
+        name="email"
+        rules={[
```

Comment:
```suggestion
        extra={errors?.email ? errors.email.join("\n") : undefined}
        rules={[
```
This will give you inline feedback from the happy explaining why this field failed.